src/toolbars/examples/toolbar.js

```diff
@@ -50,7 +50,7 @@
   <file name="index.html">
     <div ng-controller="ViewCtrl" class="row example-container">
       <div class="col-md-12">
-        <pf-toolbar id="exampleToolbar" config="toolbarConfig">
+        <pf-toolbar id="exampleToolbar" config="toolbarConfig" on-toggle-colvis="toggleColvis(column)">
          <actions>
```

Comment:
Hi @dtaylor113 , I just followed the inputs/outputs styles described in [angular guide](https://docs.angularjs.org/guide/component#component-based-application-architecture). Here, we expose the "column" variable to its parent controller scope in order to communicate with sibling component pfTableView.